.github/workflows/stale-bot.yml

```diff
@@ -0,0 +1,19 @@
+name: Mark stale issues and pull requests
+on:
+  schedule:
+    - cron: "30 1 * * *"
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          days-before-stale: 90
+          stale-issue-message: 'This issue is stale because it has been open 180 days with no activity. Remove stale label or comment or this will be closed in 7 days.'
```

Comment:
This is around 6 months?

Reply:
It's around 3mo. 180 would be closer to 6mo if that's what you want.

Reply:
I meant because of the message:
> it has been open 180 days

Reply:
Let's update the messages that get posted to reflect the actual days we're using :)

Reply:
Doh! My bad, fixed.